### DISCUSSION TITLE
error deploy vercel got error Module not found: Can't resolve 'encoding'

### DISCUSSION THREAD

**Discussion by rahulsya** [vercel/next.js#50063] [App Router] [answered]
I encounter this error when deploying next 13 on vercel 
Module not found: Can't resolve 'encoding' in '/vercel/path0/node_modules/node-fetch/lib'

I already add package `encoding `on devDeps

**Comment by ZainKhalidOfficial** [comment]
Any solution?

**Reply by rahulsya** [reply, marked as answer]
my solution is fix all warning & error , to check that you can run `npm run lint`
